submissions/evgenii-del/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,425 @@
+          <div class="months__slider">
+            <button class="months__control left" type="button">
+              <img class="months__control-img"
+                   src="https://raw.githubusercontent.com/evgenii-del/cinema-app/2f2581807cd2a5eb89131a7bd9494aef330c36db/images/arrow.svg"
+                   alt="arrow">
+            </button>
+            <ol class="months__list">
+              <li class="months__list-item">
+                <input class="months-checkbox" type="radio" name="months" id="months-1">
+                <label class="months-label" for="months-1">
+                  <span class="months__list-day-of-week">tue</span>
+                  <span class="months__list-day">3</span>
+                </label>
+              </li>
+              <li class="months__list-item">
+                <input class="months-checkbox" type="radio" name="months" id="months-2">
+                <label class="months-label" for="months-2">
+                  <span class="months__list-day-of-week">web</span>
+                  <span class="months__list-day">4</span>
+                </label>
+              </li>
+              <li class="months__list-item">
+                <input class="months-checkbox" type="radio" name="months" id="months-3" checked>
+                <label class="months-label" for="months-3">
+                  <span class="months__list-day-of-week">thu</span>
+                  <span class="months__list-day">5</span>
+                </label>
+              </li>
+              <li class="months__list-item">
+                <input class="months-checkbox" type="radio" name="months" id="months-4">
+                <label class="months-label" for="months-4">
+                  <span class="months__list-day-of-week">fri</span>
+                  <span class="months__list-day">6</span>
+                </label>
+              </li>
+              <li class="months__list-item">
+                <input class="months-checkbox" type="radio" name="months" id="months-5">
+                <label class="months-label" for="months-5">
+                  <span class="months__list-day-of-week">sat</span>
+                  <span class="months__list-day">7</span>
+                </label>
+              </li>
+            </ol>
+            <button class="months__control right" type="button">
+              <img class="months__control-img"
+                   src="https://raw.githubusercontent.com/evgenii-del/cinema-app/2f2581807cd2a5eb89131a7bd9494aef330c36db/images/arrow.svg"
+                   alt="arrow">
+            </button>
+          </div>
+        </div>
+        <div class="time">
+          <h2 class="time__title">Time</h2>
+          <ul class="time__list">
+            <li class="time__list-item">
+              <input class="time-checkbox" type="radio" name="times" id="time-1" checked>
+              <label class="time-label" for="time-1">
+                <span class="time__list-choice">15:00</span>
```

Comment:
For date and time preferable to use `<time>` tag [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)